third_party/FMI/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 
 INCLUDE_DIRECTORIES( "${CMAKE_SOURCE_DIR}/third_party/Expat/lib")
 INCLUDE_DIRECTORIES( "${CMAKE_SOURCE_DIR}/third_party/")
+INCLUDE_DIRECTORIES( ${CMAKE_SOURCE_DIR}/third_party/zlib )
```

Comment:
I moved it from the top CMakeLists.txt file. It is only used in the FMI/FMU stuff and conflicted with the zlib I'm using for the ground plot library.